packages/ibm-products/src/components/ConditionBuilder/ConditionBlock/ConditionBlock.js

```diff
@@ -0,0 +1,247 @@
+import React, { useContext, useState } from 'react';
+import { Close } from '@carbon/react/icons';
+import { ConditionBuilderItem } from '../ConditionBuilderItem/ConditionBuilderItem';
+import { pkg } from '../../../settings';
+import PropTypes from 'prop-types';
+
+import cx from 'classnames';
+import ConditionConnector from '../ConditionConnector/ConditionConnector';
+
+import { ConditionBuilderContext } from '../ConditionBuilderContext/DataTreeContext';
+import { ConditionBuilderButton } from '../ConditionBuilderButton/ConditionBuilderButton';
+
+/**
+ * This component build each block of condition consisting of property, operator value and close button.
+ */
+const blockClass = `${pkg.prefix}--condition-builder`;
```

Comment:
instead of declaring this in every component file i would just create a constants file that you can import the block class from.

Reply:
moved to config file